modules/external-secrets-enable-operator-metrics.adoc

```diff
@@ -0,0 +1,208 @@
+// Module included in the following assemblies:
+//
+// * security/external_secrets_operator/exteernal-secrets-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="external-secrets-enable-operator-metrics_{context}"]
+= Configuring metrics collection for {external-secrets-operator} by using a ServiceMonitor
+
+[role="_abstract"]
+The {external-secrets-operator} exposes metrics by default on port `8443` at the `/metrics` service endpoint. You can configure metrics collection for the Operator by creating a `ServiceMonitor` custom resource (CR) that enables the Prometheus Operator to collect custom metrics. For more information, see "Configuring user workload monitoring".
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {external-secrets-operator}.
+* You have enabled the user workload monitoring.
+
+.Procedure
+
+. Configure the Operator to use `HTTP` or `HTTPS` protocols for the metrics server:
```

Comment:
Should we mention HTTPS is enabled by default and follow step 1 for enabling HTTP instead.